domain/consensus/processes/dagtraversalmanager/window.go

```diff
@@ -45,6 +67,37 @@ func (dtm *dagTraversalManager) calculateBlockWindowHeap(stagingArea *model.Stag
 		return nil, err
 	}
 
+	_, err = dtm.daaWindowStore.DAAWindowBlock(dtm.databaseContext, stagingArea, current, 0)
+	isNotFoundError := database.IsNotFoundError(err)
+	if !isNotFoundError && err != nil {
+		return nil, err
+	}
+
+	if isNotFoundError && currentGHOSTDAGData.SelectedParent() != nil {
+		windowHeapSlice, err := dtm.windowHeapSliceStore.Get(stagingArea, highHash, windowSize)
```

Comment:
Seems like `highHash` is an error, shouldn't it be `currentGHOSTDAGData.SelectedParent()` ? (the comment two lines below assumes that as well)